src/org/labkey/test/tests/FileBasedPipelineTest.java

```diff
@@ -72,6 +73,28 @@ public void startTest()
         goToProjectHome();
     }
 
+    @Test
+    public void testScriptBasedFilewatchers()
+    {
+        String pipelineTaskWithHelpText = "Pipeline task with help text";
+        String helpText = "Hello! I am help text.";
+        String pipelineTaskNoHelpText = "two-steps";
+        String pipelineTaskInvisible = "Task that is not evokable by filewatcher";
+
+        goToFolderManagement().goToImportTab();
+        clickAndWait(Locator.lkButton("Manage file watcher triggers"));
```

Comment:
this (clicking the 'manage file watcher triggers' button) could be an opportunity to scaffold in a page object, and to implement finding and clicking the button in the ImportTabPage class.
If you don't have bandwidth or scope to do that just now, please let me know- I'd like to track this as future work.